README.md

```diff
@@ -1 +1,67 @@
-Tools for Wizzilab products
+# AppLink via python
+
+Python scripts to communicate with Wizzilab's AppLink. 
+
+- watch_reports.py allows to display reports coming from sensors
+- read_example.py allows to read a property of a device 
+- write_example.py shows how to configure a property of a device. 
+
+## Installation and example
+
+In order to use these scripts you just need to install the paho-mqtt module.
+
+```
+pip3 install paho-mqtt
+```
+
+Once this is done, it is possible to run the scripts, -h option is available to get a list of the required parameters. The watch_reports.py script requires the login credentials that can be found on the Dash7Board the company's page in the AppLink section, for example to connect to the queue with the following login credentials  
+
+| MQTT Host             | roger.wizzilab.com  |
+| --------------------- | ------------------- |
+| Port                  | 8883                |
+| User                  | xfj2l79tn5f2        |
+| Client ids            | xfj2l79tn5f2:[0-9]  | 
+| MQTT Topics           | /applink/01BC50C7/# | 
+| Key                   | password            |
+
+just run the following command: 
+
+```
+./watch_reports.py -b roger.wizzilab.com -p 8883 -c 01BC50C7 -u xfj2l79tn5f2 -pass password -id 0
+```
+
+The read_example script, which reads a property from the sensor, requires three additional parameters. The uid of the device, the file id of the file containing the searched parameter and finally the name of the desired field. A description of the available properties and their associated file ids is available in XML format at https://dash7board.wizzilab.com/devices/UID/host_xml. Here is an extract from the file https://dash7board.wizzilab.com/devices/001BC50C7001087A/host_xml.
+```xml
+<file file_id="131" forward="2" name="Sensors Report">
+    <u16 name="vcore" display_name="Core Voltage (mV)" access="r">
```

Comment:
Remove that field, it just clutters the README, and has a chance of wasting the reader's time, as it is not directly link to the command example.
Just directly start with the 3 dot ellipsis.